strawberry_django/pagination.py

```diff
@@ -81,16 +131,23 @@ def apply_pagination(
         self,
         queryset: _QS,
         pagination: Optional[object] = None,
+        *,
+        related_field_id: Optional[str] = None,
     ) -> _QS:
-        return apply(pagination, queryset)
+        return apply(pagination, queryset, related_field_id=related_field_id)
 
     def get_queryset(
         self,
         queryset: _QS,
         info: Info,
         *,
         pagination: Optional[object] = None,
+        _strawberry_related_field_id: Optional[str] = None,
```

Comment:
This is passed in `**kwargs`. Wanted to make sure this would not conflict at all with some user-given argument in the resolver.

But maybe the leading `strawberry_` is enough? What do you think?

Reply:
probably, but it is fine to keep the underscore